StellarApp/Controllers/MenuViewController.swift

```diff
@@ -1,30 +1,7 @@
-//
-//  MenuViewController.swift
-//  StellarApp
-//
-//  Created by TingxinLi on 4/8/19.
-//  Copyright © 2019 TingxinLi. All rights reserved.
-//
-
 import UIKit
 
 class MenuViewController: UIViewController {
 
```

Comment:
Same here

Reply:
yes, we will need to working on this later